src/collective/glossary/upgrades/v2/__init__.py

```diff
@@ -9,3 +9,10 @@ def update_resource_conditions(setup_tool):
     setup_tool.runImportStepFromProfile(profile, 'cssregistry')
     setup_tool.runImportStepFromProfile(profile, 'jsregistry')
     logger.info('Conditions to include package resources were updated.')
+
+
+def update_controlpanel_options(setup_tool):
+    """Update control panel options."""
+    profile = 'profile-{0}:default'.format(PROJECTNAME)
+    setup_tool.runImportStepFromProfile(profile, 'registry')
```

Comment:
@rodfersou you should use `setup_tool.runImportStepFromProfile(profile, 'plone.app.registry')`